### PR TITLE
Faris

### PR DESCRIPTION
**Pull request by FarisDianePFG**
i don't know what I did :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # WorkspaceServicesWorkshop
 workshop exercise repo
+
+modified on dev github by john
```
Comment (BakerSaraPFG) [inline README.md:4]: Looks great

**Review by BakerSaraPFG — APPROVED** [review]
Approved

File: branchExercise.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+faris line 1
+faris line 2
```
Comment (gartee-john-PFG) [inline branchExercise.txt:2]: please insert one more line after this

**Review by gartee-john-PFG — CHANGES_REQUESTED** [review]

File: commitExercise.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+line 1 dev from gartee
+line 2 dev from gartee
```
Comment (AndersonRyanPFG) [inline commitExercise.txt:2]: Please use blue color font.

**Review by AndersonRyanPFG — APPROVED** [review]
Make it beautiful

File: branchExercise.txt (hunk 2 above)
Comment (FarisDianePFG) [reply branchExercise.txt:2]: added an additional line per your request

File: commitExercise.txt (hunk 3 above)
Comment (FarisDianePFG) [reply commitExercise.txt:2]: John says no way!  not a standard.  👎

**Review by gartee-john-PFG — APPROVED** [review]
wonderful

File: README.md (hunk 1 above)
Comment (RikerGregPFG) [reply README.md:4]: Looks good!

**Review by RikerGregPFG — APPROVED** [review]

**Review by SusmanNickiPFG — APPROVED** [review]
Looks good!

**Comment by FarisDianePFG** [comment]
tried to merge into main instead of dev.  closing and trying again

**Pull request closed without merging**